pkg/apis/provisioning/v0alpha1/types.go

```diff
@@ -50,6 +50,11 @@ type GitHubRepositoryConfig struct {
 	// Whether we should show dashboard previews for pull requests.
 	// By default, this is false (i.e. we will not create previews).
 	GenerateDashboardPreviews bool `json:"generateDashboardPreviews,omitempty"`
+
+	// The prefix for the Grafana data. If specified, Grafana will ignore anything that is outside this prefix.
```

Comment:
Is this going to be only specific to Github or should it be at the repository level instead?

Reply:
I debated that -- but I think it really only makes sense in git.  For local/s3/gcs/??? it seems odd to have an additional parameter past the root/path

so I think it make sense as part of git (and other systems where the root only gets us half way)

Reply:
I think it makes sense to have to all repositories just for completion's sake, but this is one of those features that are _easiest_ to implement specifically for Git, at least with our current abstraction layers.